help/en/html/doc/Technical/gitadmin.shtml

```diff
@@ -50,11 +50,11 @@
       JMRI's Github is configured with the following automations and requirements for merging a Pull Request:
       <ul>
         <li>All CI tests have passed for the Pull Request</li>
-        <li>There has been at least one review from a JMRI developer that accepts the Pull Request</li>
+        <li>There has been at least one review from a non-author JMRI developer that accepts the Pull Request</li>
         <li>There are no reviews requesting changes</li>
         <li>The Pull Request has been in-queue for at least a day to allow people to take a look at them</li>
         <li>These rules apply to all members of the development team, including administrators</li>
-        <li>The original author of the Pull Request is automatically assigned to the Pull Request</li>
+        <li>The original author of the Pull Request has been assigned to the Pull Request</li>
```

Comment:
This is actually less technically correct than the wording I propose.  There is a GitHub automation in place that automatically assigns the originating author.